File: JAVA/ParkJaemin/java_work/src/Eighth/Homework/Homework2.java
package Eighth.Homework;

class Number2 {
    int start;
    int end;

    public Number2(int start, int end) {
        this.start = start;
        this.end = end;
        for (int i = start; i <= end; i++) {
            System.out.println(i);
        }
    }
}

public class Homework2 {
    public static void main(String[] args) {
        System.out.println("3~33까지 출력해보자");
        Number2 B = new Number2(3, 33);


    }
}

File: JAVASCRIPT/ParkJaemin/js_test/src/data_type/HomeworkReview/Homework2.js
const Homework2 = () => {
    const START = 1;
    const END = 20;


    for (var i = START; i <= END; i++) {
        if(!(i % 2)){ // i값을 2로 나눌때 떨어진다면 ㅡ> i % 2 ==0
            console.log("Homework2 : " + i)
        }
    }

    return (
        <div className="Homework2">
            <p>

                Homework2 Works Fine!

            </p>

        </div>
    )

}

export default Homework2